Package.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version:5.3
+import PackageDescription
+let package = Package(
+    name: "FreshchatSDK",
+    platforms: [
+        .iOS(.v8)
```

Comment:
```suggestion
        .iOS(.v9)
```

Hi @hemanth-sudarsan, I've integrated this version in my project (at [GetYourGuide](https://github.com/getyourguide/)) and was able to compile it on the new MacBook Pro (w/ Apple M1). But I had to make a minor change to make it compile, I had to update the minimum version to `9`. Could you confirm that and update the minimum version if possible?

Thank you!

Reply:
I would also be interested in getting this corrected so that the Xcode warning is removed (as mentioned by @T2SoftDev  a few months ago). Also, there is a 4.2.0 release available but not with SPM. How long before this release is available with the SPM beta branch?

Reply:
@hemanth-sudarsan Any update on this? You've released another version 4.3.0, but there are still no updates to spm-beta branch. Please update spm-beta branch with latest changes or finally merge spm-beta to master 🙏

Reply:
Yep, looks like this should be set to something greater than iOS 8. I get the following warning in Xcode 13:

> .../checkouts/freshchat-ios/Package.swift: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'FreshchatSDK' from project 'FreshchatSDK')


Xcode 12 dropped support for iOS 8 so its on library vendors to update their projects to reflect a more appropriate deployment target for modern build systems - https://www.jessesquires.com/blog/2020/07/20/xcode-12-drops-support-for-ios-8-fix-for-cocoapods/